ios/Classes/FSQIPBuyerVerification.m

```diff
@@ -0,0 +1,902 @@
+/*
+ Copyright 2019 Square Inc.
+ 
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+ 
+ http://www.apache.org/licenses/LICENSE-2.0
+ 
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+*/
+
```

Comment:
can't we do something like this:https://github.com/square/react-native-square-reader-sdk/blob/386f60c606af11183aa14504940c3a145cbd1f46/ios/RNReaderSDKCheckout.m#L72

This seems scary to maintain such a long list.

Reply:
I've provided the feedback to IAP team. Unfortunately such utility does not exist and they don't plan to add soon. I do think it is something that should be included in IAP SDK as developers will need it. So it's an item in improvement list I plan to send them later. But I guess this is what we have to maintain for now.